redirects.yml

```
@@ -28,10 +28,10 @@ redirects:
   'solutions/observability/apps/real-user-monitoring-rum.md': 'solutions/observability/apm/real-user-monitoring-rum.md'
   'solutions/observability/apps/create-upload-source-maps-rum.md': 'solutions/observability/apm/create-upload-source-maps-rum.md'
   'solutions/observability/apps/use-opentelemetry-with-apm.md': 'solutions/observability/apm/use-opentelemetry-with-apm.md'
-  'solutions/observability/apps/upstream-opentelemetry-collectors-language-sdks.md': 'solutions/observability/apm/upstream-opentelemetry-collectors-language-sdks.md'
-  'solutions/observability/apps/collect-metrics.md': 'solutions/observability/apm/collect-metrics.md'
-  'solutions/observability/apps/limitations.md': 'solutions/observability/apm/limitations.md'
-  'solutions/observability/apps/resource-atrributes.md': 'solutions/observability/apm/resource-attributes.md'
+  'solutions/observability/apps/upstream-opentelemetry-collectors-language-sdks.md': 'solutions/observability/apm/use-opentelemetry-with-apm.md'
+  'solutions/observability/apps/collect-metrics.md': 'solutions/observability/apm/use-opentelemetry-with-apm.md'
+  'solutions/observability/apps/limitations.md': 'solutions/observability/apm/use-opentelemetry-with-apm.md'
+  'solutions/observability/apps/resource-atrributes.md': 'solutions/observability/apm/use-opentelemetry-with-apm.md'
```

Redirects for deleted pages:

```suggestion
  'solutions/observability/apps/resource-atrributes.md': 'solutions/observability/apm/use-opentelemetry-with-apm.md'
  'solutions/observability/apm/collect-metrics.md': 'solutions/observability/apm/use-opentelemetry-with-apm.md'
  'solutions/observability/apm/limitations.md': 'solutions/observability/apm/use-opentelemetry-with-apm.md'
  'solutions/observability/apm/resource-attributes.md': 'solutions/observability/apm/use-opentelemetry-with-apm.md'
  'solutions/observability/apm/upstream-opentelemetry-collectors-language-sdks.md': 'solutions/observability/apm/use-opentelemetry-with-apm.md'
  'solutions/observability/get-started/quickstart-monitor-hosts-with-opentelemetry.md': 'solutions/observability/get-started.md'
  'solutions/observability/get-started/quickstart-unified-kubernetes-observability-with-elastic-distributions-of-opentelemetry-edot.md': 'solutions/observability/get-started.md'
```